packages/astro/src/default/components/MobileContentToggle.astro

```diff
@@ -0,0 +1,50 @@
+---
+
+---
+
+<div class="h-12 sm:hidden"></div>
+<view-toggle
+  class="fixed sm:hidden h-12 bottom-0 w-full bg-tk-elements-app-backgroundColor border-t border-tk-elements-app-borderColor flex justify-center items-center z-60 text-tk-elements-app-textColor"
+>
+  <button aria-hidden="true">Tutorial</button>
+  <button aria-pressed="true" class="rounded-full w-8 h-4 p-0.5 bg-gray-4 dark:bg-gray-6 mx-2 relative">
+    <span class="inline-block transition-all absolute top-0.5 left-0.5 rounded-full bg-white dark:bg-gray-2 w-3 h-3"
+    ></span>
+  </button>
+  <button aria-hidden="true">Editor</button>
```

Comment:
I would recommend to remove the `aria-hidden`'s from here. Right now these elements are present in tab order but excluded from accessibility tree. Assistive technologies will not read these but users will still be able to navigate there. Also the 2nd button is present in accessibility tree but doesn't have any name so it's announced just as button. And as the `aria-pressed` state is not updated, maybe remove that one too. 

There are some accessibility issues in the project at the moment so might be good to tackle all of those at once at some point.

Reply:
Yep makes sense, thanks for the feedback :pray: